public/OutlawDerby2024.md

```diff
@@ -0,0 +1,65 @@
+![outlawlogo]("C:\Users\brian\Documents\GitHub\cub306.org\public\images\outlaw derby lOGO.png")
```

Comment:
I would think this would be better to put in the events folder

Reply:
I created a outlaw folder in the 23-24 events folder and moved the .md and images to that folder